packages/synapse-react-client/src/lib/containers/auth/TwoFactorAuthSettingsPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Button, Typography, Stack, Skeleton } from '@mui/material'
+import React from 'react'
+import IconSvg from '../IconSvg'
+import { TWO_FACTOR_DOCS_LINK } from './TwoFactorEnrollmentForm'
+import {
+  useDisableTwoFactorAuth,
+  useGetTwoFactorEnrollmentStatus,
+} from '../../utils/hooks/SynapseAPI/auth/useTwoFactorEnrollment'
+import ConditionalWrapper from '../utils/ConditionalWrapper'
+import { displayToast } from '../ToastMessage'
+
+export type TwoFactorAuthSettingsPanelProps = {
+  onRegenerateBackupCodes: () => void
+  onBeginTwoFactorEnrollment: () => void
+}
+
+export default function TwoFactorAuthSettingsPanel(
+  props: TwoFactorAuthSettingsPanelProps,
+) {
+  const { onRegenerateBackupCodes, onBeginTwoFactorEnrollment } = props
+  const { data: status, isLoading: isLoadingStatus } =
+    useGetTwoFactorEnrollmentStatus()
+  const isActivated = status?.status === 'ENABLED'
+
+  const { mutate: disable2FA, isLoading: isLoadingMutation } =
+    useDisableTwoFactorAuth({
+      onSuccess: () => {
+        displayToast('2FA removed from this account', 'info')
```

Comment:
Designs use different style toast in SageAccountWeb, not sure if that will use the same API or if this should be a prop.

Reply:
I think this is fine for SageAccountWeb for now.  If design thinks this is a high priority issue, we can work it into a future sprint.